google-ads-examples/src/main/java/com/google/ads/googleads/examples/hotelads/AddHotelCallout.java

```diff
@@ -0,0 +1,231 @@
+            return;
+        }
+
+        try {
+            new AddHotelCallout()
+                .runExample(
+                    googleAdsClient,
+                    params.customerId,
+                    params.campaignId,
+                    params.calloutText,
+                    params.languageCode
+                );
+        } catch (GoogleAdsException gae) {
+            // GoogleAdsException is the base class for most exceptions thrown by an API request.
+            // Instances of this exception have a message and a GoogleAdsFailure that contains a
+            // collection of GoogleAdsErrors that indicate the underlying causes of the
+            // GoogleAdsException.
+            System.err.printf(
+                "Request ID %s failed due to GoogleAdsException. Underlying errors:%n",
+                gae.getRequestId()
+            );
+            int i = 0;
+            for (GoogleAdsError googleAdsError : gae.getGoogleAdsFailure().getErrorsList()){
+                System.err.printf("  Error %d: %s%n", i++, googleAdsError);
+            }
+        }
+    }
+
+       /**
+        * Runs the example.
+        *
+        * @param googleAdsClient the Google Ads API client.
+        * @param customerId the client customer ID.
+        * @param campaignId the campaign ID.
+        * @param calloutText the text for the callout.
+        * @param languageCode the language code for the callout (IETF BCP 47 compliant).
+        * @throws GoogleAdsException if an API request failed with one or more service errors.
+        */
+        private void runExample(
+            GoogleAdsClient googleAdsClient, 
+            long customerId, 
+            long campaignId,
+            String calloutText,
+            String languageCode) {
+
+            // Creates the extension feed item.
+            String extensionFeedItemResourceName = addExtensionFeedItem(googleAdsClient, customerId, calloutText, languageCode);
+
+            // Adds the extension feed item to the campaign.
+            addExtensionToCampaign(googleAdsClient, customerId, campaignId, extensionFeedItemResourceName);
+        }
+
+        /**
+         * Creates a new extension feed item for the callout.
+         *
+         * @param googleAdsClient the Google Ads API client.
+         * @param customerId the client customer ID.
+         * @param campaignID the campaign ID.
+         * @param calloutText the text for the callout.
+         * @param languageCode the language code for the callout (IETF BCP 47 compliant).
+         * @return resource name of the newly created extension feed item.
+         * @throws GoogleAdsException if an API request failed with one or more service errors.
+         */
+        private String addExtensionFeedItem(
+            GoogleAdsClient googleAdsClient, 
+            long customerId, 
+            String calloutText, 
+            String languageCode) {
+                // Create the callout with text and language of choice
```

Comment:
Please write comments so they complete the sentence "This thing..."

So here it would be "Creates ..." rather than "Create ..."

Reply:
+1. Same for the two objects below.